x-pack/test/functional/apps/code/with_security.ts

```diff
@@ -88,18 +90,15 @@ export default function testWithSecurity({ getService, getPageObjects }: FtrProv
       });
 
       it('only codeAdmin can manage repositories', async () => {
-        await login(codeAdmin);
-        await retry.tryForTime(5000, async () => {
-          const buttons = await testSubjects.findAll('importRepositoryButton');
-          expect(buttons).to.have.length(1);
-        });
-        await PageObjects.code.fillImportRepositoryUrlInputBox(
-          'https://github.com/elastic/TypeScript-Node-Starter'
+        await repoLoad(
+          'github.com/elastic/TypeScript-Node-Starter',
+          'typescript_node_starter',
+          config.get('kbnTestServer.installDir') || REPO_ROOT
         );
-        // Click the import repository button.
-        await PageObjects.code.clickImportRepositoryButton();
+        await esArchiver.load('code/repositories/typescript_node_starter');
 
-        await retry.tryForTime(300000, async () => {
+        {
```

Comment:
Extra block?

Reply:
just trying to differentiate 2 minor steps in this test case , which helps to be eyeballed.